### ISSUE TITLE
Request for necessary function

### ISSUE THREAD

**Issue by OrionCodeDev** [ryqndev/pick-ban-pro#1]
Hey mate,

It's really cool tool and we wanna use it in our project, but it's missing "swap" phase after all player pick champions. It's will be unique when you will add it. It's will be then #1 pick ban tool <3

Best Regards

**Comment by ryqndev** [comment]
tl;dr: At the end of the day, if a coach wants to send their completed draft to another person for analysis, the pick order is what's most important, rather than the entire comp as a whole. 

The ability to swap is something I've considered but not extremely high priority atm. Since the point of this *tool* is to provide high level analysis in drafting phase, the resulting pick/ban **order** is what matters the most - rather than simulate the entire champion selection phase (also the reason why the ability to select summoner spells is not a part of this tool). If I were to add swaps, I'd first implement the ability to "swap out" or "edit" picks in the middle or after draft to help coaches visualize what another comp would look like if they had selected something differently. 

something to consider: The only benefit to being able to swap picks is to show what champs will end up in what roles - something coaches/players should be able to deduce if they're at a high enough level to care about tournament draft with the sole exception of flex picks. 

Thanks for the interest in the project!

**Comment by ryqndev** [comment]
If you really need this feature, you can reach out to me about how you would exactly integrate this app with your own and I can see what work would need to be done.
